packages/sysutils/busybox/scripts/dtfile

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+COMPATIBLE=$(cat /proc/device-tree/compatible | tr -d '\000' | sed -n -e 's/.*\(allwinner\|amlogic\|rockchip\).*/\1/p')
```

Comment:
`cat /proc/device-tree/compatible 2>/dev/null`

And same for other script too, such as dispensing with DTFILE/DTNAME variable and just outputting the result of the `grep` or `cat` command.